lib/mozart_fetcher/components/envelope.ex

```diff
@@ -0,0 +1,7 @@
+defmodule MozartFetcher.Components.Envelope do
```

Comment:
not sure how I feel about having `MozartFetcher.Envelope`, and `MozartFetcher.Components.Envelope` modules, maybe its completely fine, just starting the conversation :)

Reply:
Yeah this is one of things I wanted to think about changing. Components.EnvelopeComponent felt unnecessary but calling them both Envelope is also confusing..

Reply:
Solved by not using structs for Components. Good suggestion! :)